nextflow_schema.json

```
@@ -405,7 +405,22 @@
                         "plasmidfinder",
                         "resfinder",
                         "vfdb"
```

Looking at their code, seems that there are more options for dbs (I was trying to find the documentation, actually). Not sure if they are relevant, but just for completeness:
  
```suggestion
                        "vfdb",
                        "megares",
                        "ecoli_vf",
                        "ecoh",
                        "bacmet2",
                        "victors"
 
```